api/remotefirewaller/remotefirewaller.go

```diff
@@ -26,6 +26,11 @@ func NewClient(caller base.APICallCloser) *Client {
 	return &Client{ClientFacade: frontend, facade: backend}
 }
 
+// Close implements io.Close
+func (c *Client) Close() error {
```

Comment:
Nice pickup, there were other places where this had been done; I've fix them all